### DISCUSSION TITLE
M401A网卡驱动问题

### DISCUSSION THREAD

**Discussion by RagBagCN** [ophub/amlogic-s9xxx-armbian#1606] [Amlogic]
**Device Information | 设备信息**
 - SOC: [ S095L3A]
 - Model [M401A]

**Armbian Version | 系统版本**
 - Kernel Version: [e.g. 5.4.252_server_2023.08.09]
 - Release: [bullseye]

已将”Armbian_23.08.0_amlogic_s905l3a_bullseye_5.4.252_server_2023.08.09.img.gz“写入EMMC，将盒子已经硬改板载网卡rtl8822cs，系统正常，但输入”nmtui“找不到无线网卡选项

**Comment by ophub** [comment]
https://github.com/unifreq/linux-6.1.y/tree/main/drivers/net/wireless/realtek/rtl88x2cs

https://github.com/unifreq/linux-5.15.y/tree/main/drivers/net/wireless/realtek/rtl88x2cs

https://github.com/unifreq/linux-5.10.y/tree/main/drivers/net/wireless/realtek/rtl88x2cs

https://github.com/unifreq/linux-5.4.y/tree/main/drivers/net/wireless/realtek/rtl88x2cs

四个系列的内核里都有rtl8822cs的驱动，但是这是从第三方引入的，有人说能用，也有人说不能用，自己换内核测试下哪个能用吧。

**Reply by RagBagCN** [reply]
好的我试一下，感谢大佬
